frontend/packages/dev-console/console-extensions.json

```diff
@@ -85,6 +85,59 @@
       ]
     }
   },
+  {
+    "type": "console.cluster-configuration/item",
+    "properties": {
+      "id": "dev-console.WebTerminalConfiguration",
```

Comment:
As written above, this code and extension should be moved into the `console-app` `console-extensions.json` as well